src/Agent/NewRelic/Agent/Core/Configuration/DefaultConfiguration.cs

```diff
@@ -2131,7 +2131,81 @@ public bool AiMonitoringRecordContentEnabled
 
         public Func<string, string, int> LlmTokenCountingCallback => _runTimeConfiguration.LlmTokenCountingCallback;
 
-        public bool AzureFunctionModeEnabled => _bootstrapConfiguration.AzureFunctionModeEnabled;
+        #region Azure function support
+
+        public bool AzureFunctionModeDetected => _bootstrapConfiguration.AzureFunctionModeDetected;
+
+        private bool? _azureFunctionModeEnabled;
+        public bool AzureFunctionModeEnabled =>
+            _azureFunctionModeEnabled ??
+            (_azureFunctionModeEnabled = EnvironmentOverrides(TryGetAppSettingAsBoolWithDefault("AzureFunctionModeEnabled", false), "NEW_RELIC_AZURE_FUNCTION_MODE_ENABLED")).Value;
+
+        public string AzureFunctionResourceUri
```

Comment:
This will be needed later I'm guessing?

Reply:
Good catch, that was actually supposed to be the implementation for `AzureFunctionResourceId` - making this property unnecessary.